KeeAgent/UI/EntryPanel.cs

```diff
@@ -135,8 +135,13 @@ void UpdateKeyInfo()
 
       try {
         var key = CurrentSettings.TryGetSshPublicKey(pwEntryForm.EntryBinaries);
+        var comment = key.Comment;
+        if (string.IsNullOrEmpty(comment)) {
+          var privateKey = CurrentSettings.GetSshKey(pwEntryForm.EntryBinaries, pwEntryForm.EntryRef);

```

Comment:
Since this decrypts the private key, this can be a very long running blocking operation. We can't have this here for performance reasons. (And we don't want to decrypt the private key unless we are actually using the key for signing.)

Reply:
Hmm, I didn't think this method would be called often.
But I think the comment is an important part of the key, to know what the key is.
Do you have another idea to achieve it? ex.. caching?

Reply:
The workaround is to provide a separate public key file that contains the comment.

Reply:
Another feature that I considered it to have something like a "Test" button that will decrypt the private key to ensure that your passphrase is correct. Such a feature could also show the encrypted comment when the test was performed.

Reply:
Hmm. If so, what is the natural behavior of this UI?
I'm a bit confused about the current behavior, comment textbox is empty, but the agent shows a comment.

I consider a disabled textbox, but we don't distinguish between an empty comment and no comment field. I think that a disabled textbox is not suitable for an empty comment.

Reply:
> Another feature that I considered it to have something like a "Test" button that will decrypt the private key to ensure that your passphrase is correct. Such a feature could also show the encrypted comment when the test was performed.

I didn't notice before replying. I agreed to it. but unfortunately, I don't have enough time to implement that feature.
I'll leave it as-is for this part.

Reply:
For now, perhaps we could show "<encrypted>" in the comment box if the private key uses the openssh key file format an no separate public key was given?